components/layout_2020/flow/float.rs

```diff
@@ -81,29 +81,42 @@ pub(crate) struct PlacementAmongFloats<'a> {
     /// The next band, needed to know the height of the last band in current_bands.
     next_band: FloatBand,
     /// The size of the object to place.
-    object_size: Vec2<Length>,
+    object_size: &'a Vec2<Length>,
     /// The minimum position in the block direction for the placement. Objects should not
     /// be placed before this point.
     ceiling: Length,
+    /// The inline position where the object would be if there were no floats. The object
+    /// can be placed after it due to floats, but not before it.
+    min_inline_start: Length,
+    /// The maximum inline position that the object can attain when avoiding floats.
+    max_inline_end: Length,
 }
 
 impl<'a> PlacementAmongFloats<'a> {
     pub(crate) fn new(
         float_context: &'a FloatContext,
         ceiling: Length,
-        object_size: Vec2<Length>,
```

Comment:
I think it makes sense to simply clone the input to this function instead of passing a reference.

Reply:
Done